app/controllers/users/edit_password_controller.rb

```diff
@@ -23,15 +23,26 @@ def update
     private
 
     def user_params
-      params.require(:update_user_password_form).permit(:password, :current_password)
+      params.require(:update_user_password_form).permit(:password)
     end
 
     def handle_success
+      re_encrypt_active_profile
+
```

Comment:
I am interested in *why* are are adding extra newlines like this to methods in some parts of this app. Is there a rule of thumb for when are are doing it?